anvio/dbops.py

```diff
@@ -2573,7 +2574,7 @@ def get_gene_level_coverage_stats_entry_for_inseq(self, gene_callers_id, split_c
 
         # INSEQ/Tn-SEQ views
         mean_coverage = 0
-        detection = numpy.count_nonzero(gene_coverage_values_per_nt) / gene_length
+        #detection = numpy.count_nonzero(gene_coverage_values_per_nt) / gene_length
```

Comment:
so if I am not wrong, this is the part where you are calculating the detection value of a gene in INSEQ/Tn-SEQ results, right?

In metagenomic read recruitment studies it makes a lot of sense to talk about 'detection' as a function of the number of positions in the gene that is covered by at least 1X (as described [here](http://merenlab.org/2017/05/08/anvio-views/#detection)). But in the case of INSEQ/Tn-SEQ, I think detection statistic should behave differently so the user can immediately see genes that recruit no reads at all. For this, one could imagine setting the detection value to 1.0 even if 5% of the positions in the gene are covered. I'm not sure if I'm making any sense, but I wanted to add this as a comment.